executor/common.h

```diff
@@ -402,6 +417,13 @@ static uint16 csum_inet_digest(struct csum_inet* csum)
 
 #if SYZ_EXECUTOR || __NR_syz_execute_func
 // syz_execute_func(text ptr[in, text[taget]])
+#if GOOS_windows
+static long syz_execute_func(long text)
```

Comment:
I think it's better to add `defined(__GNUC__)` to `#if GOARCH_amd64` below. It will result in less duplication.
It's what you want, right?

Reply:
I'm thinking of changing the function **syz_execute_func** to this 
```
#if defined(__GNUC__)
	volatile long p[8] = {0};
	(void)p;
#if GOARCH_amd64
	asm volatile("" ::"r"(0l), "r"(1l), "r"(2l), "r"(3l), "r"(4l), "r"(5l), "r"(6l),
		     "r"(7l), "r"(8l), "r"(9l), "r"(10l), "r"(11l), "r"(12l), "r"(13l));
#endif
#endif
	((void (*)(void))(text))();
	return 0;
```
This way there is no duplication of code.

Reply:
I would combine both ifdef because... well, the fewer ifdefs, the better.